scaffolding/lib/copyCommand.js

```diff
@@ -14,8 +14,8 @@ const
 
 function injectTemplates(sdkInfos, src, dest) {
   const longDescriptionRegexp = {
-    start: `\# ${_.camelCase(sdkInfos.action)}\n`,
-    end: '\#\# Signature\n'
+    start: `# ${_.camelCase(sdkInfos.action)}\n`,
+    end: '## Signature\n'
```

Comment:
If you remove the backslash, `#` will be interpreted when constructing a new regexp instead of matching the `#` character.
Same apply for the following declarations

Reply:
As far as I recall, `#` is not a special character. And I can't find anything about this in either [Javascript RegExp documentation](https://devdocs.io/javascript/global_objects/regexp), nor in the [PCRE one](https://perldoc.perl.org/perlre.html)

Moreover, `\#` is not a valid escape sequence.

Reply:
You're right! Didn't know why I thought that..